elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -92,6 +92,14 @@
             <version>${hibernate5.version}</version>
         </dependency>
 
+        <!-- Caffeine cache -->
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>2.8.4</version>
```

Comment:
should this dependency be optional?

Reply:
The default configuration now uses Caffeine, so if we make it optional, won't that force the average user, who isn't specifying an alternative, to declare an extra dependency?